src/utils/tasks.js

```diff
@@ -0,0 +1,62 @@
+import TaskState from '@/model/TaskState.model'
+
+/**
+ * States used when comparing the parent with children nodes.
+ * @type {*[]}
+ */
+const orderedStates = [
```

Comment:
Does the `TaskState` enum allow us to order the task states to avoid defining here?

Reply:
Yup. When the `TaskState` enum is created, the array given to the `initEnum` function is saved to a read-only `Array`. Iterating the enum, is the same as iterating that array, so the values will be returned in order (tested [here](https://github.com/kinow/notebooks/blob/master/javascript/enumify/Enumify.ipynb)).

So we could remove `orderedStates`, but would have to keep `isStoppedOrderedStates`. Changing, added the following note to where the enum is created, so other devs will remember that maintaining the order of entries created in the enum is important:

```js
// NOTE: the order of the enum values is important to calculate the group states in the UI. Items at
// the top of the list have preference over the items below in the algorithm.
```